src/components/mainContent/ForwardButton.js

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect } from "react";
+import "./mainContent.css";
+import arrows from "../../assets/upArrow.svg";
+
+export default function ForwardButton(props) {
+  let currentSection = props.content[props.visibleSections];
+  let currentLength = undefined;
+  if (currentSection && currentSection.post)
+    currentLength = currentSection.post.bodyText.length;
+  else if (currentSection && currentSection.followups)
+    currentLength = currentSection.followups.length;
+
+  function addVisibleText() {
+    if (props.currentVisibleText < currentLength) {
+      handleForward();
+      console.log("Added");
+    } else props.setForwardVisible(true);
+  }
+
+  useEffect(() => {
+    let forwardSection = setInterval(() => addVisibleText(), 1000);
+    return function cleanup() {
```

Comment:
This function doesn't need to be named, it can be anonymous

Reply:
The cleanup or the interval?